gh-actions/go/code-sanity/action.yaml

```diff
@@ -67,9 +67,13 @@ runs:
           name=$1
           path=$2
           version="$(
-            go mod edit --json | jq -r ".Require[] | select(.Path==\"${path}\") | .Version" || echo 'latest'
+            go mod edit --json | jq -r ".Require[] | select(.Path==\"${path}\") | .Version"
           )"
-          echo "${name}=${version}" >> $GITHUB_OUTPUT
+          if [ -z "${version}" ]; then
```

Comment:
I think should that would be more readable btw:
```sh
version="latest"
mod_version="$(
            go mod edit --json | jq -r ".Require[] | select(.Path==\"${path}\") | .Version"
)"
if [ -n "${mod_version}" ]; then
   version="${mod_version}"
fi
echo "${name}=${version}" >> $GITHUB_OUTPUT
```

with:
- default
- If override -> reassign
- action

Reply:
Sounds good